### PR TITLE
Some fixes for radmin tests

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: src/tests/radmin/all.mk [hunk 1]
```diff
@@ -40,8 +40,8 @@ $(OUTPUT)/%: $(DIR)/% test.radmin.radiusd_kill test.radmin.radiusd_start
 		echo "--------------------------------------------------"; \
 		echo "$(RADIUSD_RUN)"; \
 		echo "$(RADMIN_BIN) -q -f $(RADMIN_SOCKET_FILE) > $(FOUND) < $<"; \
-		$(MAKE) $(TEST).radiusd_kill; \
-		exit 1;\
+		$(MAKE) test.radmin.radiusd_kill; \
```
Comment (alandekok) [inline src/tests/radmin/all.mk]: why is this preferred?  I'm happy to leave it as $(TEST)

Comment (jpereira) [reply src/tests/radmin/all.mk]: I would like too. but, does not expand.

```
[jpereira@sugarloaf freeradius-server.git]$ make Q= test.radmin | grep radiusd_kill
if [ -f build/tests/radmin/radiusd.pid ]; then ret=0; if ! ps `cat build/tests/radmin/radiusd.pid` >/dev/null 2>&1; then rm -f control-socket; echo "FreeRADIUS terminated during test called by test.radmin.radiusd_kill"; echo "GDB output was:"; cat "build/tests/radmin/gdb.log"; echo "--------------------------------------------------"; tail -n 40 "build/tests/radmin/gdb.log"; echo "Last entries in server log (build/tests/radmin/gdb.log):"; ret=1; fi; if ! kill -TERM `cat build/tests/radmin/radiusd.pid` >/dev/null 2>&1; then ret=1; fi; exit ; fi
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
```

Comment (jpereira) [reply src/tests/radmin/all.mk]: Even I've tried to use double reference but without success. e.g: `$$(TEST).radiusd_kill`

Comment (alandekok) [reply src/tests/radmin/all.mk]: Ah OK.  More boilermake magic.

File: src/tests/radmin/all.mk [hunk 2]
```diff
@@ -54,6 +54,5 @@ $(OUTPUT)/%: $(DIR)/% test.radmin.radiusd_kill test.radmin.radiusd_start
 		touch $@;\
 	fi
 
-$(TEST): $(FILES)
+$(TEST): ${BUILD_DIR}/bin/radmin $(FILES)
```
Comment (alandekok) [inline src/tests/radmin/all.mk]: That should just edit the build rules, like src/tests/unit/all.mk:

```
#
#	Run the radmin commands against the radiusd.
#
$(OUTPUT)/%: ${BUILD_DIR}/bin/radmin $(DIR)/% test.radmin.radiusd_kill test.radmin.radiusd_start
```

Comment (jpereira) [reply src/tests/radmin/all.mk]: done

**Pull request merged**